lib/common/utils.js

```diff
@@ -13,6 +13,15 @@ export const waitFor = function waitFor(ms) {
   return new Promise(resolve => setTimeout(resolve, ms || 0))
 }
 
+export const isString = function isString(obj) {
+  return typeof obj === 'string' || obj instanceof String
+}
+export const startsWithAlias = aliasArray => str => aliasArray.some(c => str.indexOf(c) === 0)
```

Comment:
```js
export const startsWithAlias = aliasArray => str =>
  aliasArray.some(al => str.slice(0, al.length) === al)
```
Otherwise it will continue to match the rest of the string, only to then be rejected by the `===0`

Reply:
JUST KIDDING, this clearly calls for REGULAR EXPRESSIONS ™
```js
const regExEscapeEx = /[-\/\\^$*+?.()|[\]{}]/g
const escapeRegex = str => str.replace(regExEscapeEx, "\\$&")
export const startsWithAlias  = aliasArray => {
  const ex = new RegExp(`^(:?${aliasArray.map(escapeRegex).join("|")})`)
  return str => ex.test(str)
}
```
**Note:** This is serious, please include it.

Reply:
Would rather go with `startsWith`